dask/README.md

```diff
@@ -0,0 +1,80 @@
+# Dask
+
+[Dask](https://dask.org/) is an open source set of tools for parallelizing Python analytics tasks. It is developed in coordination with popular open source Python libraries including Numpy, Pandas and Scikit-Learn. 
+
+This initialization action will create a Conda environment bundled with Dask and [Dask-Yarn](https://yarn.dask.org/), a tool for running Dask jobs on a YARN cluster such as [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster. You can then take advantage of Dataproc's features such as scaling out jobs with [Autoscaling](https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/autoscaling).
```

Comment:
I think that RAPIDS init action also supports Dask, but in standalone mode, not on YARN. May we extract creation of Dask cluster from RAPIDS init action to this init action and add an option that allows users to chose what type of Dask installation users want to use (YARN or Standalone)?

Reply:
Ack. I can add standalone to this init action but was planning to modify the RAPIDS init action in a future PR. Should I just bundle it here?

Reply:
No, I agree that it's better to split them. Also we will need to let NVIDIA know about RAPIDS init action changes, so they can update their tutorials too.

Reply:
Agreed, sounds good!